docs/extras/fastapi.md

```diff
@@ -11,3 +11,15 @@ The FastAPI extra node can be used by adding the following to `pyproject.toml`
 ## API Reference
 
 ::: aineko.extras.fastapi.main.FastAPI
+
+# Health Check Endpoint
+
+Aineko comes with a light weight helper function as follows:
+
+::: aineko.extras.fastapi.health.read_health
+
+## Authentication Considerations
+
+By default, the Aineko health endpoint does not contain authentication. 
+
+One method to add authentication is to use security middleware at the app level for FastAPI. This will handle adding authentication to the health endpoint without having to add custom dependencies.
```

Comment:
Maybe we should link the FastAPI docs here: https://fastapi.tiangolo.com/advanced/security/

Reply:
updated here: [42dc84c](https://github.com/aineko-dev/aineko/pull/142/commits/42dc84c8995f32e0b2a788e40c2903ee5a29eb10)